.devcontainer/devcontainer.json

```diff
@@ -15,8 +15,8 @@
         "ghcr.io/devcontainers/features/common-utils:2": {},
         "ghcr.io/devcontainers/features/dotnet:2": {
             "version": "none",
-            "dotnetRuntimeVersions": "7.0",
-            "aspNetCoreRuntimeVersions": "7.0"
+            "dotnetRuntimeVersions": "9.0",
```

Comment:
This isn't really needed since the SDK image will already have them.  This actually should say `8.0` for both these values to bring in ALSO the net8 runtime which the extensions rely on.  This will make the image be 'ready' state more for the VSCode extensions.

Reply:
got it!